.travis.yml

```diff
@@ -240,8 +240,9 @@ jobs:
 
     # Run Coverity
     - stage: Test different OS/CXX/Flags
+      if: type = cron
       os: linux
-      sudo: required
+      sudo: false
```

Comment:
I'm arguably too late to the party, but how is this compatible with the actual use of `sudo` below?